root/static/scripts/common/components/CommonsImage.js

```diff
@@ -36,7 +36,7 @@ class CommonsImage extends React.Component<Props, State> {
     }
   }
 
-  render() {
+  render(): React.MixedElement | null {
```

Comment:
Why not `React.Element<'div'> | null` instead?

Reply:
That fact that it's a div (instead of a span or any other element) didn't seem important, and that type information isn't used anywhere.